plugins/drill-coverage-plugin/src/agentPartMain/kotlin/CoverageAgentPart.kt

```diff
@@ -41,17 +42,25 @@ class CoveragePlugin(override val id: String) : InstrumentedPlugin<CoverConfig,
     }
 
     override fun initPlugin() {
+        val str = "Plugin $id initialized!"
+        val message = JSON.stringify(
+            CoverageMessage.serializer(),
+            CoverageMessage(CoverageEventType.INIT, str)
+        )
+        Sender.sendMessage("coverage", message)
+        println(str)
     }
 
-    override fun instrument(className: String, initialBytest: ByteArray): ByteArray {
-        try {
-            initialClassBytes[className] = initialBytest
-            sendClass(ClassBytes(className, initialBytest.toList()))
-            return instrumenter(className, initialBytest)
-        } catch (ex: Throwable) {
-            ex.printStackTrace()
-            throw ex
-        }
+    override fun instrument(className: String, initialBytes: ByteArray): ByteArray {
+        return if ("_\$\$_" !in className && "CGLIB\$\$" !in className) {
```

Comment:
Why did we skip these classes? There are a lot of ways to do javaProxies. We don't have any chance to predict this. We should extract this to config.

Reply:
This is a quick solution for the demo. Let's do the config feature in the future.